### PR TITLE
Added helper to improve readability of pointer literals

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Added helper to improve readability of pointer literals

### CODE REVIEW

**Review by ykim-1 — APPROVED** [review]
LGTM, tests pass locally

File: pointer_helpers.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package linodego
+
+func Pointer[T any](value T) *T {
```
Comment (yec-akamai) [inline pointer_helpers.go:3]: Can we have some notes here to explain why and how to use the `Pointer`? Maybe add an example as well.

Comment (ezilber-akamai) [reply pointer_helpers.go:23]: Good suggestion, thanks!

**Review by yec-akamai — APPROVED** [review]
Nice work! 🎉

**Pull request merged**